packages/apps/fortune/exchange-oracle/server/src/common/interfaces/job.ts

```diff
@@ -0,0 +1,5 @@
+export interface ISolution {
```

Comment:
Solution can like 
{
  workerAddress: string;
  solution: string;
}

And then have like:
export interface ISolutionsFile {
    exchangeAddress: string;
    solutions: ISolution[];
}

So the file content would the class ISolutionsFile